.changeset/glorious-turtles-try.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
@tylerbutler, do we control name generation? They are funny, but I'm not sure they belong in repo.

Reply:
They're random and meaningless. Chosen randomly to avoid merge collisions. They can be renamed; the names are not needed. They could be GUIDs.

Reply:
I'd probably go with guids. Feels more professional :)

Reply:
More info here: https://github.com/changesets/changesets/blob/main/docs/common-questions.md#each-changeset-is-its-own-file